blazor/rich-text-editor/getting-started.md

```diff
@@ -51,7 +51,7 @@ Open **~/_Imports.razor** file and import the Syncfusion.Blazor namespace.
 
 Now, register the Syncfusion Blazor Service in the Blazor Server App or Blazor WebAssembly App. Here, Syncfusion Blazor Service is registered by setting [IgnoreScriptIsolation](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.GlobalOptions.html#Syncfusion_Blazor_GlobalOptions_IgnoreScriptIsolation) property as true to load the scripts externally in the [next steps](#add-script-reference).
 
-> From 2022 Vol1 (20.1) version - The default value of `IgnoreScriptIsolation` is changed as `true`, so, you don’t have to set `IgnoreScriptIsolation` property explicitly to refer scripts externally.
+> From 2022 Vol1 (20.1) version - The default value of `IgnoreScriptIsolation` is changed as true, so, you don’t have to set `IgnoreScriptIsolation` property explicitly to refer scripts externally.
```

Comment:
use as `true` here, since we need to highlight it.

Reply:
Review comments addressed sir